runtime/rasdump/javadump.cpp

```diff
@@ -5854,27 +5872,42 @@ spaceIteratorCallback(J9JavaVM* virtualMachine, J9MM_IterateSpaceDescriptor* spa
 
 	jcw->_OutputStream.writeCharacters("NULL\n");
 	jcw->_OutputStream.writeCharacters("1STHEAPTOTAL   ");
-	jcw->_OutputStream.writeCharacters("Total memory:        ");
+	jcw->_OutputStream.writeCharacters("Total memory:          ");
 	jcw->_OutputStream.writeVPrintf(FORMAT_SIZE_DECIMAL, decimalLength, sizeTotal);
 	jcw->_OutputStream.writeCharacters(" (");
 	jcw->_OutputStream.writeVPrintf(FORMAT_SIZE_HEX, sizeof(void *) * 2, sizeTotal);
 	jcw->_OutputStream.writeCharacters(")\n");
 	if (0 != sizeTarget) {
 		jcw->_OutputStream.writeCharacters("1STHEAPTARGET  ");
-		jcw->_OutputStream.writeCharacters("Target memory:       ");
+		jcw->_OutputStream.writeCharacters("Target memory:         ");
 		jcw->_OutputStream.writeVPrintf(FORMAT_SIZE_DECIMAL, decimalLength, sizeTarget);
 		jcw->_OutputStream.writeCharacters(" (");
 		jcw->_OutputStream.writeVPrintf(FORMAT_SIZE_HEX, sizeof(void *) * 2, sizeTarget);
 		jcw->_OutputStream.writeCharacters(")\n");
 	}
 	jcw->_OutputStream.writeCharacters("1STHEAPINUSE   ");
-	jcw->_OutputStream.writeCharacters("Total memory in use: ");
+	jcw->_OutputStream.writeCharacters("Total memory in use:   ");
 	jcw->_OutputStream.writeVPrintf(FORMAT_SIZE_DECIMAL, decimalLength, allocTotal);
 	jcw->_OutputStream.writeCharacters(" (");
 	jcw->_OutputStream.writeVPrintf(FORMAT_SIZE_HEX, sizeof(void *) * 2, allocTotal);
 	jcw->_OutputStream.writeCharacters(")\n");
+	if (virtualMachine->memoryManagerFunctions->j9gc_off_heap_allocation_enabled(virtualMachine)) {
+		jcw->_OutputStream.writeCharacters("2STHEAPSPCUSE  ");
+		jcw->_OutputStream.writeCharacters(spaceDescriptor->name);
+		jcw->_OutputStream.writeCharacters(" memory in use: ");
```

Comment:
I think this needs three more spaces after the colon so the values written by lines 5898 and 5904 will line up.

Reply:
Hm, not clear why. Note that I use '+ 7 - strlen(spaceDescriptor->name' to account for the differencies between the strlen of 'Off-heap' and 'Flat'.

I can use strlen("Off-heap") instead to 7, so it's more clear.